### PR TITLE
mimir.rules.kubernetes component: add RBAC information to the docs

### PR DESCRIPTION
**Pull request by ptodev**
Quick update to the documentation for `mimir.rules.kubernetes` following a discussion with @Logiraptor

### CODE REVIEW

**Review by Logiraptor — APPROVED** [review]
Looks great! left one minor typo correction

**Comment by rfratto** [comment]
@Logiraptor Do you think we should update the Helm chart so these RBAC rules are applied by default?

**Comment by Logiraptor** [comment]
@rfratto That sounds like a good idea! Probably should keep it as a separate role and role binding so it's easy to update or disable individually later if that ever became an issue.

File: docs/sources/flow/reference/components/mimir.rules.kubernetes.md [hunk 1]
```diff
@@ -209,4 +216,39 @@ mimir.rules.kubernetes "default" {
         }
     }
 }
-```
\ No newline at end of file
+```
+
+Below is an example RBAC configuration for Kubernetes - it authorizes the Agent to query the Kubernetes REST API:
```
Comment (karengermond) [inline docs/sources/flow/reference/components/mimir.rules.kubernetes.md:221]:
```suggestion
The following example is an RBAC configuration for Kubernetes. It authorizes the Agent to query the Kubernetes REST API:
```

**Review by karengermond — APPROVED** [review]
LGTM

**Comment by ptodev** [comment]
Hi @robert, please feel free to merge it whenever you like. I don't have permissions yet 😄

**Pull request merged**